### ISSUE TITLE
Quiet period always getting overwritten to index < 5 ? 0 : 2 * 60 everytime a job/pipeline is edited and saved

### ISSUE THREAD

**Issue by kannanprasanna** [jenkinsci/tikal-multijob-plugin#212]
There have been some many issues reported earlier and PRs submitted related to this already. But all we need is to remove the default section (default="index &lt; 5 ? 0 : 2 * 60") out of  https://github.com/jenkinsci/tikal-multijob-plugin/blob/dc55c20dc01fb435d269c37319e99b21318f9f50/src/main/resources/com/tikal/jenkins/plugins/multijob/MultiJobBuilder/config.jelly#L89 . Currently, we couldn't set this to 0 or any other value since it gets overwritten again with default value when any job is edited. Could this be fixed and released soon, please?

**Comment by wdougoveto** [comment]
I have confirmed this issue as well. +1

**Comment by yorammi** [comment]
@kannanprasanna @wdougoveto @cloudyster - is this can be fixed by PR #201 that fixes issue #200? If so, please close this as a duplicate of #200, and I'll merge the PR and soon will release a new version

**Comment by cloudyster** [comment]
PR #201 fixes this issue as well as #200; additionally it avoids quiet time calculation for disabled jobs

**Comment by kannanprasanna** [comment]
I agree with @cloudyster. I'll close this issue now @yorammi . Eagerly awaiting the next release!

**Comment by wdougoveto** [comment]
@yorammi, thank you for getting version 1.34 merged and tagged. Do you know when will this start to show up as a new version in the jenkins plugins?

**Comment by yorammi** [comment]
I have an issue with the jenkins registry when uploading the new version.
once solved it willbe released

בתאריך יום ב׳, 19 באוק׳ 2020, 17:18, מאת wdougoveto ‏<
notifications@github.com>:

**Comment by wdougoveto** [comment]
Thank you!

**Comment by turnopil** [comment]
Any updates so far? I can't see new release

**Comment by kannanprasanna** [comment]
@yorammi , could you please let us know when the next versions could be released?

**Comment by wdougoveto** [comment]
@yorammi, also wondering when the next version will be available?

**Comment by kannanprasanna** [comment]
@yorammi , thanks for getting version 1.36 published. It works fine!
